### DISCUSSION TITLE
Payment method page design for website

### DISCUSSION THREAD

**Discussion by Sapacezonebd** [community/community#145661] [Programming Help]
### Select Topic Area

Question

### Body

Payment method page design for website

**Comment by ebndev** [comment]
Hi @Sapacezonebd, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there. If you provide more details on what you're looking for, that might help someone give you a nudge in the right direction. 😁
